src/style.scss

```diff
@@ -0,0 +1,127 @@
+$main-color: #060b35 ;
+$secondary-color: #616070 ;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", "Manrope", sans-serif;
+  color: $main-color;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  background-color: #fff;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    width: 160px ;
+    height: 134px ;
+    margin: 0 auto;
+    margin-bottom: 40px;
+
+  }
+
+  &__name {
+    font-weight: 500;
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-weight: 400;
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    width: 166px ;
+    height: 40px ;
+    display: flex;
+    justify-content: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    background-color: #00acdc;
+    color: white;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+  }
+
+  &__buy:hover {
+    background-color: #f3f3f3;
+    color: #00acdc;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__review-box {
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+}
+
+.reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(./images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:not(:first-child) {
+    margin-left: 4px;
+  }
+
+  &--4 :nth-child(-n+4) {
+    background-image: url(./images/star-active.svg);
+  }
```

Comment:
```suggestion
  &--1 :nth-child(-n+1)
  &--2 :nth-child(-n+2)
  &--3 :nth-child(-n+3),
  &--4 :nth-child(-n+4),
  &--5 :nth-child(-n+5) {
    background-image: url(./images/star-active.svg);
  }
```
not implemented this point from task description.
![Screenshot from 2023-06-25 16-28-16](https://github.com/mate-academy/layout_product-cards/assets/33029864/0161a557-6ea3-48ae-b5b6-437d41a4d17a)